### ISSUE TITLE
Change subtitle output 'auto' to whatever language is detected

### ISSUE THREAD

**Issue by heldheld** [ralosant/FlowLauncher.Google.Translator#2]
Let's take the word 'chair' as the query as an example.

So right now, when you use the autodetect feature, the output subtitle shows:
[auto: chair].

It would be nice if it showed the autodetected language, so for example:
[en: chair].

Or maybe:
[auto en: chair]

Since some words can be the same in different languages, it's good to know which language was actually autodetected.

**Comment by ralosant** [comment]
I completely agree, but without fully changing the workflow of the plugin should not be possible. 
Google URL (as used here) or in case Google API can not directly retrieve the auto detected language as far as I read in the documentation.
So for the moment I don´t know an easy way to implement it, if you have some idea it will be welcome.

**Comment by heldheld** [comment]
I had already looked at it with my very limited knowledge before posting to see if I could figure it out, but unfortunately I couldn't.

**Comment by jerrygreen** [comment]
Characters have their unicode numbers. It is possible to tell the language from just a bunch of characters.

Even if they look the same, for example:

- [Unicode Character “a” (U+0061)](https://www.compart.com/en/unicode/U+0061)
- [Unicode Character “а” (U+0430)](https://www.compart.com/en/unicode/U+0430)

Like these, chars have their own different code, even if they look the same.

In some languages which use the same code, they often have their own unique characters.

So, from a string it's possible to tell the language – programmatically.

While it might be easy to hardcode a couple languages like this, of course it's harder if you want to support all the languages.

Although, I don't know if there's a good lib for python to easily determine language from chars, as I'm not a python developer.
